dapp-core/src/test/resources/sample_contract.groovy

```diff
@@ -0,0 +1,39 @@
+import io.reactivex.Observable
+import iroha.protocol.Commands
+import jp.co.soramitsu.dapp.AbstractDappScript
+import jp.co.soramitsu.dapp.helper.CacheManager
+import jp.co.soramitsu.iroha.java.IrohaAPI
+import jp.co.soramitsu.iroha.java.Transaction
+
+import java.security.KeyPair
+
+class TestContract extends AbstractDappScript {
+
+    private Observable<Commands.Command> observable
+
+    TestContract(IrohaAPI irohaAPI, KeyPair keyPair, CacheManager cacheManager) {
+        super(irohaAPI, keyPair, cacheManager)
```

Comment:
May you implement it without forcing the developer to call super constructor? Making life easier for smart contracts developers.

Reply:
What are the advantages of the approach? How to set those 3 values in a gentle way? In my opinion it would lead to only 1 line removal but general complication of instantiation and processing

Reply:
Discussed in a meeting. We should avoid showing any logic that a smart contract developer should not affect.